setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-__version__ = "1.5.0"
+__version__ = "1.5.1"
```

Comment:
Please don't update the version in a feature PR. We'll do that during a release.